packages/desktop-client/src/components/settings/BudgetTypeSettings.tsx

```diff
@@ -21,35 +20,42 @@ export function BudgetTypeSettings() {
     <Setting
       primaryAction={
         <Button onPress={onSwitchType}>
-          Switch to {budgetType === 'report' ? 'envelope' : 'tracking'}{' '}
-          budgeting
+          <Trans>
+            Switch to{' '}
+            {{ budgetType: budgetType === 'report' ? 'envelope' : 'tracking' }}{' '}
```

Comment:
Unfortunately this sort of conditional can be confusing to translators as it doesn't show which word goes inside. Maybe we can conditionally render two different `<Trans>` instead?

Reply:
Probably best, as envelope/tracking isn't translated either so it would look daft in other languages